quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java

```diff
@@ -105,6 +109,20 @@ public static PropertyMapper[] getHttpPropertyMappers() {
         };
     }
 
+    private static Optional<String> validatePath(Optional<String> value, ConfigSourceInterceptorContext context) {
```

Comment:
Just a minor though, it might be more direct to express this as:

```
    private static BiFunction<Optional<String>, ConfigSourceInterceptorContext, Optional<String>> validatePath(String key) {
        return (value, context) -> value.filter(v -> v.equals(context.proceed(key).getValue())).map(p -> p.replace("\\", "/"));
    }
```

Reply:
@shawkins addressed, thanks!